tasks/build.yml

```diff
@@ -1,28 +1,37 @@
 ---
 
-- name: Download GPG key
-  command: curl -LO https://deb.nodesource.com/gpgkey/nodesource.gpg.key
-  args:
-    chdir: /tmp
-    creates: /tmp/nodesource.gpg.key
+- name: Install Python Modules
+  become: yes
+  pip:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - pip==9.0.3
```

Comment:
I feel like the versions should be configurable, why have these modules been added anywho? To get around SSL errors?

Reply:
`pip` and `setuptools` of at least those versions are required to install `ndg-httpsclient`, which in turn is required to validate https://deb.nodesource.com/

Reply:
I have change it to "greater than", which should be better and not overwrite modules already installed, as long as they have a minimum version.  As this role has nothing directly to do with python, and only requires this to validate the nodejs repo, I feel python modules shouldn't be configurable in the role.